filer/templates/admin/filer/folder/directory_listing.html

```diff
@@ -137,17 +137,37 @@
 
         {% block object-tools %}
             <div class="navigator-tools">
+                {% if actions_selection_counter %}
+                    <script type="text/javascript">var _actions_icnt="{{ paginated_items.object_list|length|default:"0" }}";</script>
+                    <span class="action-counter">{{ selection_note }}</span>
+                    {% if paginator.count != paginated_items.object_list|length %}
+                    <span class="all">{{ selection_note_all }}</span>
+                    <span class="question">
+                        <a href="javascript:;" title="{% trans "Click here to select the objects across all pages" %}">{% blocktrans with paginator.count as total_count %}Select all {{ total_count }} files and/or folders{% endblocktrans %}</a>
```

Comment:
why the `href="javascript:;"` here? Maybe just use a regular `#` as a href and do the `event.preventDefault()`. This click is handled somewhere in js anyway, I think.